snuba/query/processors/empty_tag_condition_processor.py

```diff
@@ -50,6 +50,12 @@ def process_condition(exp: Expression) -> Expression:
                     if exp.function_name == ConditionFunctions.EQ:
                         replacement = FunctionCall(exp.alias, "not", (replacement,))
 
+                    prev_value = query.get_experiment_value(
```

Comment:
give context in the comment and then it LGTM

Reply:
Updated.